tests_e2e/tests/multi_config_ext/multi_config_ext.py

```diff
@@ -36,7 +38,7 @@
 
 class MultiConfigExt(AgentVmTest):
     class TestCase:
-        def __init__(self, extension: VirtualMachineExtensionClient, get_settings: Callable[[str], Dict[str, str]]):
+        def __init__(self, extension: AzureSdkClient, get_settings: Callable[[str], Dict[str, str]]):
```

Comment:
why superclass and not concrete class type? do we pass different subclasses of azuresdkclient?